### PR TITLE
Email template fix

### PR DESCRIPTION
**Pull request by technabors**
This PR simply redoes the layout for the email template page. No content edits were performed aside from including a H2 title Introduction.

### CODE REVIEW

**Comment by technabors** [comment]
This corresponds to issues/340

**Comment by technabors** [comment]
Invaded this PR to hide sliders.md file in 
453b794.

**Comment by nimbinatus** [comment]
@technabors, try using the keyword in the PR description to get the issue to autoclose when this merges: https://help.github.com/articles/closing-issues-using-keywords/

**Comment by technabors** [comment]
Added the keywords to auto-close.

File: _docs/templates/email-template.md [hunk 1]
```diff
@@ -21,32 +26,56 @@ if more information is needed links to control panels are provided.
...
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
 ## Typography and image guides
 
 {% figure [caption:"Quick guide for typography and image specifications"] [class:"image bg-light border"] %}
-  ![]({{site.cdn_url}}/img/layout/email-templates/Email%20Detail%20-%20Spec.svg){:width=60%}
+  ![]({{site.cdn_url}}/img/layout/email-templates/Email%20Detail%20-%20Spec.svg)
```
Comment (nimbinatus) [inline _docs/templates/email-template.md]: This one looks a little odd in places, too.
screen shot 2017-11-20 at 11 00 08 am
screen shot 2017-11-20 at 11 00 37 am

File: _docs/templates/email-template.md [hunk 2]
```diff
@@ -66,56 +95,86 @@ followed by `.rackspace.com`.
...
     - Remote Observations
 
 {% endcolumn %}
 
-{% column right:"hxCol-md-8" %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
 {% figure [caption:""] [class:"image bg-light border"] %}
-  ![]({{site.cdn_url}}/img/layout/email-templates/Urgent%20Alert.svg){:width=60%}
+  ![]({{site.cdn_url}}/img/layout/email-templates/Urgent%20Alert.svg)
```
Comment (nimbinatus) [inline _docs/templates/email-template.md]: Maybe it's something with FF57 (Quantum)? I don't remember them looking weird last week...

Spacing off:
screen shot 2017-11-20 at 11 02 02 am

s/`dispurtion`/`disruption`
screen shot 2017-11-20 at 11 03 12 am

File: _docs/templates/email-template.md [hunk 3]
```diff
@@ -66,56 +95,86 @@ followed by `.rackspace.com`.
...
 {% endcolumn %}
+
 </div>
 
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
 {% figure [caption:"Alert banner typography, color, and spacing"] [class:"image bg-light border"] %}
   ![]({{site.cdn_url}}/img/layout/email-templates/Alert%20Specs.svg){:width=60%}
```
Comment (nimbinatus) [inline _docs/templates/email-template.md]: More spacing oddities (`Robo to`, `#   FAFAFA`) and spelling fix needed:
screen shot 2017-11-20 at 11 04 52 am

Also, can we drop the width modifier, or is it needed here?

File: _docs/templates/email-template.md [hunk 4]
```diff
@@ -126,25 +185,35 @@ message it is. The header contains three parts:
 
 {% endcolumn %}
 
-{% column right:"hxCol-md-8" %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
 {% figure [caption:"Visual example of Sub-Header and spacing"] [class:"image bg-light border"] %}
-  ![]({{site.cdn_url}}/img/layout/email-templates/Header.svg){:width=60%}
+  ![]({{site.cdn_url}}/img/layout/email-templates/Header.svg)
```
Comment (nimbinatus) [inline _docs/templates/email-template.md]: What's the spacing between the platform name and the message type?
screen shot 2017-11-20 at 11 06 38 am

File: _docs/templates/email-template.md [hunk 5]
```diff
@@ -126,25 +185,35 @@ message it is. The header contains three parts:
 
 {% endcolumn %}
 
-{% column right:"hxCol-md-8" %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
 {% figure [caption:"Visual example of Sub-Header and spacing"] [class:"image bg-light border"] %}
```
Comment (nimbinatus) [inline _docs/templates/email-template.md]: s/`Sub-header`/`header`

File: _docs/templates/email-template.md [hunk 6]
```diff
@@ -155,11 +224,12 @@ sent. This includes notification types and ticket or notification statuses.
...
```
Comment (nimbinatus) [inline _docs/templates/email-template.md]: s/`Sub-header`/`sub-header`

File: _docs/templates/email-template.md [hunk 7]
```diff
@@ -126,25 +185,35 @@ message it is. The header contains three parts:
...
 
 To give the user more context about the message they are about to read the
 subheading will let them know what kind of notification or ticket has been
 sent. This includes notification types and ticket or notification statuses.
 
 <div class="hxRow"  markdown="1">
-{% column left:"hxCol-md-4" %}
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
 -   **Notifications** - Notification type: Monitoring Alerts, Billing,
```
Comment (nimbinatus) [inline _docs/templates/email-template.md]:
```markdown
- **Notification types:** `Monitoring Alert`, `Billing`, `Maintenance`, etc.
- **Ticket statuses:** `New`, `Update`, `Closed`, `Pending Customer`, etc.
- **Maintenance statuses:** `Started`, `Scheduled`, `Canceled`, or `Completed`
```

File: _docs/templates/email-template.md [hunk 8]
```diff
@@ -155,11 +224,12 @@ sent. This includes notification types and ticket or notification statuses.
 
 {% endcolumn %}
 
-{% column right:"hxCol-md-8" %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
 {% figure [caption:"Visual example of Sub-Header and spacing"] [class:"image bg-light border"] %}
   ![]({{site.cdn_url}}/img/layout/email-templates/Subheader.svg){:width=60%}
```
Comment (nimbinatus) [inline _docs/templates/email-template.md]: Remove width unless it's explicitly needed.

File: _docs/templates/email-template.md [hunk 9]
```diff
@@ -211,56 +278,75 @@ the user to scan for pertinent information quickly.
 
 {% endcolumn %}
 
-{% column right:"hxCol-md-8" %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
 {% figure [caption:"Call Out box used for workarounds and comments. Comment includes code block."] [class:"image bg-light border"] %}
-  ![]({{site.cdn_url}}/img/layout/email-templates/Comment%20with%20CodeBlock.svg){:width=60%}
+  ![]({{site.cdn_url}}/img/layout/email-templates/Comment%20with%20CodeBlock.svg)
```
Comment (nimbinatus) [inline _docs/templates/email-template.md]: screen shot 2017-11-20 at 11 12 46 am

File: _docs/templates/email-template.md [hunk 10]
```diff
@@ -211,56 +278,75 @@ the user to scan for pertinent information quickly.
 
 {% endcolumn %}
 
-{% column right:"hxCol-md-8" %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
 {% figure [caption:"Call Out box used for workarounds and comments. Comment includes code block."] [class:"image bg-light border"] %}
```
Comment (nimbinatus) [inline _docs/templates/email-template.md]: s/`Call Out box`/`Callout box`

File: _docs/templates/email-template.md [hunk 11]
```diff
@@ -176,19 +246,16 @@ sent. This includes notification types and ticket or notification statuses.
   Regards, Best, etc. If the email is a ticket, it is ok to include Racker name
   and title.
 
-<!--<figure class="image">
-          <img src="{{site.cdn_url}}/img/layout/email-templates/Mobile%20LInks.svg">
-      </figure>
--->
-
 ### Call outs
```
Comment (nimbinatus) [inline _docs/templates/email-template.md]: global search/replace: s/`Call outs`/`Callouts`

File: _docs/templates/email-template.md [hunk 12]
```diff
@@ -176,19 +246,16 @@ sent. This includes notification types and ticket or notification statuses.
...
 When there is some action the user needs to take it should be prominently
 displayed and stand out from the rest of the email content. Callouts will allow
 the user to scan for pertinent information quickly.
 
 <div class="hxRow"  markdown="1">
-{% column left:"hxCol-md-4" %}
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+
 -   **Required Action** - required actions consist of tickets that are waiting
```
Comment (nimbinatus) [inline _docs/templates/email-template.md]: Capitalize `r` in `required` after the hyphen.

File: _docs/templates/email-template.md [hunk 13]
```diff
@@ -211,56 +278,75 @@ the user to scan for pertinent information quickly.
...
 button styles, and text should be in all caps. Go to the [Buttons page](http://design.rax.io/design-system/components/buttons.html)
 for in-depth use cases and standards.
 
 {% endcolumn %}
 
-{% column right:"hxCol-md-8" %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
 {% figure [caption:"Button and spacing specifications"] [class:"image bg-light border"] %}
-  ![]({{site.cdn_url}}/img/layout/email-templates/Button.svg){:width=60%}
+  ![]({{site.cdn_url}}/img/layout/email-templates/Button.svg)
```
Comment (nimbinatus) [inline _docs/templates/email-template.md]: This image is really big. Any chance it can get resized?

File: _docs/templates/email-template.md [hunk 14]
```diff
@@ -211,56 +278,75 @@ the user to scan for pertinent information quickly.
...
+{% endcolumn %}
+</div>
+
 <div class="hxRow"  markdown="1">
-{% column left:"hxCol-md-4" %}
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
 They should never break up the content of an email, and there should never be
 more than one button in the email. When adding buttons only use the standard
 button styles, and text should be in all caps. Go to the [Buttons page](http://design.rax.io/design-system/components/buttons.html)
```
Comment (nimbinatus) [inline _docs/templates/email-template.md]: Update link to `[Buttons page]({{site.url}}/components/buttons.html)`.

File: _docs/templates/email-template.md [hunk 15]
```diff
@@ -270,17 +356,19 @@ message type. It should match the content in the header of the email.
 
 {% endcolumn %}
 
-{% column right:"hxCol-md-8" %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
 {% figure [caption:"Mobile app promotion and download links"] [class:"image bg-light border"] %}
-  ![]({{site.cdn_url}}/img/layout/email-templates/Mobile%20LInks.svg){:width=60%}
+  ![]({{site.cdn_url}}/img/layout/email-templates/Mobile%20LInks.svg)
```
Comment (nimbinatus) [inline _docs/templates/email-template.md]: s/`prodcut`/`product`
screen shot 2017-11-20 at 11 17 07 am

File: _docs/templates/email-template.md [hunk 16]
```diff
@@ -379,8 +486,17 @@ For now, though, here are a few of the completed studies:
...
+</div>
+
 ## Visual examples
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
 
 {% figure [caption:"Example of a billing email using these guidelines"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/layout/email-templates/Email%20Detail.svg){:width="60%"}
+![]({{site.cdn_url}}/img/layout/email-templates/Email%20Detail.svg)
```
Comment (nimbinatus) [inline _docs/templates/email-template.md]: screen shot 2017-11-20 at 11 18 32 am

Aside from the spacing, check spelling (`prodcut` again, `tickts`)

File: _docs/templates/email-template.md [hunk 17]
```diff
@@ -21,32 +26,56 @@ if more information is needed links to control panels are provided.
...
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+## Quick view template and variable guide
 
 {% figure [caption:"Quick guide for template and variable options"] [class:"image bg-light border"] %}
-  ![]({{site.cdn_url}}/img/layout/email-templates/EmailTemplate.svg){:width=60%}
+  ![]({{site.cdn_url}}/img/layout/email-templates/EmailTemplate.svg)
```
[the remark this answers is not in the document]
Comment (technabors) [reply _docs/templates/email-template.md]: adding `@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,500');` into svg file to attempt to alleviate this issue. If not I will need to remake this file once I find the source asset.

File: _docs/templates/email-template.md (hunk 1 above)
Comment (technabors) [reply _docs/templates/email-template.md]: Need to remake this image. This is the current image that is on the live site as well. The difference between live and staging was the entire layout was borked on staging and good on live. Both used the same images however, so any issues with the images will exist in this updated branch and live.

File: _docs/templates/email-template.md (hunk 14 above)
Comment (technabors) [reply _docs/templates/email-template.md]: updated.

File: _docs/templates/email-template.md (hunk 13 above)
Comment (technabors) [reply _docs/templates/email-template.md]: I will rebuild this image.

File: _docs/templates/email-template.md (hunk 15 above)
Comment (technabors) [reply _docs/templates/email-template.md]: I will need to rebuild this image.

File: _docs/templates/email-template.md (hunk 10 above)
Comment (technabors) [reply _docs/templates/email-template.md]: updated.

File: _docs/templates/email-template.md (hunk 12 above)
Comment (technabors) [reply _docs/templates/email-template.md]: updated.

File: _docs/templates/email-template.md (hunk 11 above)
Comment (technabors) [reply _docs/templates/email-template.md]: updated.

File: _docs/templates/email-template.md (hunk 7 above)
Comment (technabors) [reply _docs/templates/email-template.md]: updated

File: _docs/templates/email-template.md (hunk 5 above)
Comment (technabors) [reply _docs/templates/email-template.md]: updated.

**Comment by nimbinatus** [comment]
Documenting decision: SVGs need to be rebuilt, so this is going to staging to at least fix the page layout before launch.

**Pull request merged**